docs/src/components/FigmaLink.tsx

```diff
@@ -1,39 +1,44 @@
 import React from 'react';
+
+import { Box } from '@marigold/components';
+
 import { Link } from './Link';
 
 export interface FigmaLinksProps {
   to: string;
 }
 export const FigmaLink = ({ to }: FigmaLinksProps) => (
-  <Link to={to} variant="outlineLink" target="_blank">
-    <svg
-      width={10}
-      height={15}
-      viewBox="0 0 10 15"
-      fill="none"
-      xmlns="http://www.w3.org/2000/svg"
-    >
-      <path
-        d="M2.36064 14.1639C3.66371 14.1639 4.72128 13.1063 4.72128 11.8033V9.44263H2.36064C1.05757 9.44263 0 10.5002 0 11.8033C0 13.1063 1.05757 14.1639 2.36064 14.1639Z"
-        fill="#1AD285"
-      />
-      <path
-        d="M0 7.08183C0 5.77876 1.05757 4.72119 2.36064 4.72119H4.72128V9.44247H2.36064C1.05757 9.44247 0 8.3849 0 7.08183Z"
-        fill="#7F43FF"
-      />
-      <path
-        d="M0 2.36064C0 1.05757 1.05757 0 2.36064 0H4.72128V4.72128H2.36064C1.05757 4.72128 0 3.66371 0 2.36064Z"
-        fill="#C00045"
-      />
-      <path
-        d="M4.72125 0H7.08189C8.38497 0 9.44253 1.05757 9.44253 2.36064C9.44253 3.66371 8.38497 4.72128 7.08189 4.72128H4.72125V0Z"
-        fill="#FF5252"
-      />
-      <path
-        d="M9.44253 7.08183C9.44253 8.3849 8.38497 9.44247 7.08189 9.44247C5.77882 9.44247 4.72125 8.3849 4.72125 7.08183C4.72125 5.77876 5.77882 4.72119 7.08189 4.72119C8.38497 4.72119 9.44253 5.77876 9.44253 7.08183Z"
-        fill="#00BBF5"
-      />
-    </svg>
-    View in Figma
-  </Link>
+  <Box py="small">
```

Comment:
Why? 🤔

Reply:
looks not good without space between. How would you handle this?